aten/src/ATen/native/Convolution.cpp

```diff
@@ -1790,15 +1794,19 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> _convolution_backward_nogroup_bac
 //       transposed is true
 //   groups: number of groups for grouped convolution
 //   output_mask: 3-dim boolean array specifying which gradients to compute in input, weight, bias order
-std::tuple<Tensor, Tensor, Tensor> convolution_backward(
+std::tuple<Tensor, Tensor, Tensor> convolution_backward_symint(
     const Tensor& grad_output_, const Tensor& input_, const Tensor& weight_,
-    const at::OptionalIntArrayRef bias_sizes_opt,
+    const at::OptionalSymIntArrayRef sym_bias_sizes_opt,
     IntArrayRef stride, IntArrayRef padding, IntArrayRef dilation, bool transposed, IntArrayRef output_padding,
     int64_t groups, std::array<bool, 3> output_mask) {
   auto grad_output = grad_output_;
   auto input = input_;
   auto weight = weight_;
 
+  c10::optional<at::IntArrayRef> int_bias_sizes_opt = sym_bias_sizes_opt ? c10::make_optional(c10::asIntArrayRefSlow(*sym_bias_sizes_opt)) : c10::nullopt;
+  at::OptionalIntArrayRef bias_sizes_opt(int_bias_sizes_opt);
```

Comment:
This is a memory error, also we should just properly symintify this function

Reply:
Wait, it's not even clear why you symint'ified this function, it's dispatched and you registered a meta instead

Reply:
oh it's because there's some path that uses convolution backward's formula which needs to be changed to `convolution_backward_symint` so we could hit the meta kernel I added in another PR